app/src/main/java/org/mozilla/focus/home/FeatureSurveyViewHelper.kt

```diff
@@ -88,7 +95,10 @@ class FeatureSurveyViewHelper internal constructor(private val context: Context,
                     // Open Play Store in a new tab
                     val url = AppConfigWrapper.getVpnRecommenderUrl(context)
                     if (!TextUtils.isEmpty(url)) {
-                        ScreenNavigator.get(context).showBrowserScreen(url, true, false)
+                        // fallback to open play store link using WebView
+                        if (!IntentUtils.handleExternalUri(context, url)) {
```

Comment:
Is this really a fallback? It looks like to me that this is the only way to go to the intended link.